viewer/packages/camera-manager/src/CameraManager.ts

```diff
@@ -0,0 +1,534 @@
+/*!
+ * Copyright 2021 Cognite AS
+ */
+
+import * as THREE from 'three';
+import TWEEN from '@tweenjs/tween.js';
+import { ComboControls } from './ComboControls';
+import { CallbackData, CameraControlsOptions } from './types';
+import { assertNever } from '@reveal/utilities';
+
+const DefaultCameraControlsOptions: Required<CameraControlsOptions> = {
+  zoomToCursor: 'basicLerp',
+  onClickTargetChange: false,
+  canInterruptAnimations: false
+};
+
+export class CameraManager extends THREE.EventDispatcher {
+  public controls: ComboControls;
```

Comment:
I don't think this should be public. Also add `readonly`

Reply:
Using it in the viewer to access controls methods, so I would say that it is reasonable to have them public

Reply:
Finally done